common/src/main/webapp/sass/explore.scss

```diff
@@ -1648,17 +1682,6 @@ table.explore-border-table {
   }
 }
 
-.p-component.p-button.pl-tiny {
```

Comment:
I guess these are now on lucuma-ui

Reply:
Yes. The lucuma-ui classes work for this now.